app/backend/src/couchers/jobs/handlers.py

```diff
@@ -484,6 +501,42 @@ def send_reference_reminders(payload):
 send_reference_reminders.SCHEDULE = timedelta(hours=1)
 
 
+def send_host_request_reminders(payload):
+    with session_scope() as session:
+        host_has_sent_message = select(1).where(
+            Message.conversation_id == HostRequest.conversation_id, Message.author_id == HostRequest.host_user_id
+        )
+
+        requests = session.execute(
+            select(HostRequest)
+            .where(HostRequest.status == HostRequestStatus.pending)
+            .where(HostRequest.host_sent_request_reminders < HOST_REQUEST_MAX_REMINDERS)
+            .where(HostRequest.start_time > func.now())
+            .where((now() - HostRequest.last_sent_request_reminder_time) >= HOST_REQUEST_REMINDER_INTERVAL)
+            .where(~exists(host_has_sent_message))
+        ).all()
+
+        for (host_request,) in requests:
+            host_request.host_sent_request_reminders += 1
+            host_request.last_sent_request_reminder_time = now()
+
+            context = SimpleNamespace(user_id=host_request.host.id)
+            notify(
+                session,
+                user_id=host_request.host.id,
+                topic_action="host_request:reminder",
+                data=notification_data_pb2.HostRequestReminder(
+                    host_request=host_request_to_pb(host_request, session, context),
+                    host=user_model_to_pb(host_request.host, session, context),
+                    surfer=user_model_to_pb(host_request.surfer, session, context),
+                ),
+            )
+
```

Comment:
Put a `session.commit()` here (a SQL `COMMIT`), so if something happens, it'll at least the ones that succeeded.